src/charm.lua

```diff
@@ -1191,13 +1229,78 @@ if next(SMODS.find_mod 'Cryptid') then
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Common", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
         end
     })
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Uncommon", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
+        end
+    })
+end
+
+if next(SMODS.find_mod 'GARBPACK') then -- Garbshit
+    Bakery_API.credit(Bakery_API.Charm {
+        key = 'Virus',
+        pos = {
+            x = 4,
+            y = 2
+        },
+        atlas = 'Charms',
+        artist = 'Jack5',
+        coder = 'Jack5',
+        unlocked = false,
+        -- TODO: Fix "ERROR" displaying in right pane when unlocking
+        locked_loc_vars = function(info_queue, card)
+            return {
+                vars = { 6 }
```

Comment:
I prefer that charm unlock conditions are generally quite hard to achieve. For this in particular, I personally think it'd be more flavorful to require that *every* remaining card is infected (which isn't *too* hard to pull off and isn't impossible to recover from).